src/ServiceControl.Config/Xaml/Controls/ExtendTrialHyperlink.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace ServiceControl.Config.Xaml.Controls
+{
+    using System;
+    using System.Diagnostics;
+    using System.Windows;
+    using System.Windows.Documents;
+
+    public class ExtendTrialHyperlink : Hyperlink
+    {
+        const string ExtendTrialUrl = "https://particular.net/extend-your-trial";
+
+        public ExtendTrialHyperlink()
+        {
+            NavigateUri = new Uri(ExtendTrialUrl);
+            Inlines.Add(ExtendTrialUrl);
+
+            Style = new Style(GetType(), FindResource(typeof(Hyperlink)) as Style);
+        }
+
+        protected override void OnClick()
```

Comment:
Could use existing command instead of custom OnClick. The command 'OpenURLCommand' does this already, all you need to do is to pass in the URL and bind it to the user action (Command property?)